docs/terminal/brisbane.md

```diff
@@ -111,8 +111,8 @@ Due to the low level of CTA (`A015`) in the AF CTR when **AF ADC** is offline, i
 ### CG ADC Offline
 Due to the low level of CTA (`A035`) in the CG CTR when **CG ADC** is offline, it is best practice to give airways clearance to aircraft at the holding point, to ensure departing aircraft can have uninterrupted climb.
 
-!!! phraseology 
-    **JST446**: "Brisbane Approach, JST446, A320, IFR, Taxiing YBCG for YSSY, Runway 14"   
+!!! phraseology
+    **JST446**: "Brisbane Approach, JST446, A320, IFR, Taxiing YBCG for YSSY, Runway 214"   
```

Comment:
I'm guessing my fix of "Runway 214" wasn't meant to have come back in?

Reply:
Good pick up, probably just a merge conflict. I'll fix it now